### DISCUSSION TITLE
Promotions

### DISCUSSION THREAD

**Discussion by lupeaserban** [vendurehq/vendure#2815] [Q&A]
Hi,

I am using a promotion action for absolute discounts on an order by implementing PromotionOrderAction. However, i noticed the absolute amount gets divided by items and then added back together. In this case, if I enter 15 Euros discount amount, after the promotion action is executed, i receive 15.01 in the active order. What am i doing wrong? Is there something i'm missing?

Thanks ahead

**Comment by michaelbromley** [comment]
Hi,
This is due to the way we handle order-level pricing by pro-rating across order lines, which can lead to rounding errors like this. Please look into the docs on MoneyStrategy, where you can choose to store money to more precision which can resolve such issues: https://docs.vendure.io/guides/core-concepts/money/#advanced-configuration-moneystrategy
